gradle-plugin/plugin/build.gradle.kts

```diff
@@ -126,12 +145,14 @@ dependencies {
   implementation(libs.kotlinx.serialization)
   implementation(libs.okhttp)
 
+  testImplementation(libs.slf4j.api)
   testImplementation(libs.google.truth)
   testImplementation(libs.junit)
   testImplementation(libs.junit.jupiter)
   testImplementation(libs.junit.jupiter.api)
   testRuntimeOnly(libs.junit.jupiter.engine)
   testRuntimeOnly(libs.junit.platform.launcher)
+  testRuntimeOnly(gradleApi())
```

Comment:
Same thing here, tests were failing at runtime when they weren’t finding gradle classes. Not sure why this wasn’t happening before. I have a feeling the shadow plugin might be messing with stuff.